protocol/optyfi-protocol/risk-framework.md

```diff
@@ -12,11 +12,11 @@ Currently the OptyFi Protocol implements three pool ratings:
 
 The system is flexible to accommodate an arbitrary number of pool ratings without modification to smart contract code. 
 
-Each liquidity pool integrated with the OptyFi Protocol is assigned a pool rating. This rating is currently established by governance vote. However in the future, governance may vote to use an external service \(such as [defiscore.io](www.defiscore.io)\), or subscribe to pool ratings from community [risk analysts](../participants.md#risk-analysts).  
+Each liquidity pool integrated with the OptyFi Protocol is assigned a pool rating. This rating is currently established through governance voting. However, in the future, governance voting may result in the use of external services \(such as [defiscore.io](www.defiscore.io)\) or community [risk analysts](../participants.md#risk-analysts) to determine the risk of a pool.  
```

Comment:
Minor feedback here to improve clarity.
Recommendations:
Each integrated liquidity pool is assigned a pool rating. 
governance voting may **decide for the** use of....
....[risk analysts](../participants.md#risk-analysts) as a determinant of the risk attached to a pool.